vignettes/teal-data-reproducibility.Rmd

```diff
@@ -10,34 +12,50 @@ vignette: >
 
 # Reproducibility of `teal_data` objects
 
-Reproducibility is a primary function of the `qenv` class, which `teal_data` inherits from. Every data modification in `teal_data` object is performed in an encapsulated environment, separate from the global environment.
+Reproducibility is a primary function of the `qenv` class, which `teal_data` inherits from.
+Every data modification in a `teal_data` object is performed in an encapsulated environment, separate from the global environment.
 
-It is important to note that the reproducibility of this object is limited only to the data-code relationship. Other aspects such as the reliability of the data source, reproducibility of the R session (including package versions), and creation and use of objects from other environments (e.g. `.GlobalEnvironment`) cannot be verified properly by the `teal_data`.
+It is important to note that the reproducibility of this object is limited only to the data-code relationship.
+Other aspects such as the reliability of the data source, reproducibility of the R session (including package versions), and creation and use of objects from other environments (e.g. `.GlobalEnv`) cannot be verified properly by `teal_data`.
+It is advisable to always begin analysis in a new session and run all code that pertains to the analysis within the `teal_data` object.
 
-## Verification status
+## Verification
+
+### Verification status
+
+Every `teal_data` object has a _verification status_, which is a statement of whether the contents of the `env` can be reproduced by `code`.
```

Comment:
Not mandatory. field or slot? But this is minor
```suggestion
Every `teal_data` object has a _verification status_ field, which is a statement of whether the contents of the `env` can be reproduced by `code`.

Also, should we use `@env` and `@code` instead of `env` and `code` ? In this case `env` is ambiguous, but if you put `@env` this at least can bring some light on the fact it's a part of `teal_object`
```

Reply:
I wanted to avoid both, actually, because we don't talk about slots in the other vignette. Perhaps that is a mistake, I do recall slots being mentioned in `teal` vignettes.

Reply:
Ah sorry. This is explained in Getting Started / Introduction vignette. So no need to put `@` before slots